WORKSPACE

```diff
@@ -16,29 +16,42 @@
 #
 
 load(
-    "//:repositories.bzl",
-    "boringssl_repositories",
-    "protobuf_repositories",
-    "googletest_repositories",
+    "//contrib/endpoints:repositories.bzl",
+    "mixer_client_repositories",
+    "servicecontrol_client_repositories",
 )
 
-boringssl_repositories()
-
-protobuf_repositories()
-
-googletest_repositories()
+mixer_client_repositories()
+
+# Workaround for Bazel > 0.4.0 since it needs newer protobuf.bzl from:
+# https://github.com/google/protobuf/pull/2246
+# Do not use this git_repository for anything else than protobuf.bzl
+new_git_repository(
+    name = "protobuf_bzl",
+    # Injecting an empty BUILD file to prevent using any build target
+    build_file_content = "",
+    commit = "05090726144b6e632c50f47720ff51049bfcbef6",
+    remote = "https://github.com/google/protobuf.git",
+)
 
 load(
-    "//contrib/endpoints:repositories.bzl",
+    "@mixerclient_git//:repositories.bzl",
```

Comment:
load as little as possible from `@mixerclient_git//:repositories.bzl`

BoringSSL has to be compatible with the version required from envoy, mixerclient has little thing to do with version compatibilities. same for grpc/protobuf/googletest/googleapis.